packages/lib/modules/pool/actions/add-liquidity/AddLiquidityProvider.tsx

```diff
@@ -41,6 +41,8 @@ export function _useAddLiquidity(urlTxHash?: Hash) {
   const [acceptPoolRisks, setAcceptPoolRisks] = useState(false)
   const [wethIsEth, setWethIsEth] = useState(false)
   const [totalUSDValue, setTotalUSDValue] = useState('0')
+  // Used when the user explicitly choses proportional input mode
+  const [wantsProportional, setWantsProportional] = useState(false)
```

Comment:
I kept the original wantsProportional naming. Maybe there's a better term but for me it clicks.